ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-3.0.yaml

```diff
@@ -153,13 +153,61 @@ tests:
           memory: 100Mi
       timeout: 2h0m0s
     workflow: servicemesh-istio-e2e-hypershift
-- as: integration-ocp
+- as: integration-ocp-latest-release
```

Comment:
Do you think it's better to name these jobs after the version?

e.g. `integration-ocp-4.14` and `integration-ocp-4.15`?

Benefits is that it's very straightforward to know which version if running just by looking at the GitHub PR. Plus, if we ever need to test on another version, using numbers is easier than `latest`, `next`, etc.